spec/classroom_spec.rb

```diff
@@ -0,0 +1,29 @@
+require_relative '../classroom'
+require_relative '../student'
+
+describe Classroom do
+  let(:classroom) { Classroom.new('A1') }
+  let(:student) { Student.new('John', 'Doe', 'john.doe@example.com', '12345') }
+
+  describe '#initialize' do
+    it 'should set the classroom label' do
+      expect(classroom.label).to eq 'A1'
+    end
+
+    it 'should initialize an empty list of students' do
+      expect(classroom.students).to be_empty
+    end
+  end
+
+  describe '#add_student' do
```

Comment:
- [OPTIONAL] Instead of using ```describe``` here, you can use ```context```. ```Describe``` block is used to define the test suite and ```context``` is used to define the various scenarios within the test suite as you are doing here.